tsconfig.json

```diff
@@ -56,6 +56,7 @@
     "node_modules/**/*",
     "packages/schematics/*/*/*files/**/*",
     "tmp/**/*",
+    "scripts/**/*",
```

Comment:
`scripts/patches/**/*`. Seems like we want to ignore patches, but still compile and lint scripts.

Reply:
Done.